docker-build.sh

```diff
@@ -11,8 +11,8 @@ hostname | tee -a /etc/ckan-cloud-operator-build-info
   curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add - && \
   apt-get update -y && apt-get install -y google-cloud-sdk kubectl postgresql nano dnsutils &&\
   echo == helm, tiller &&\
-  wget -q https://storage.googleapis.com/kubernetes-helm/helm-v2.13.1-linux-amd64.tar.gz &&\
-  tar -xzf helm-v2.13.1-linux-amd64.tar.gz &&\
+  wget -q https://get.helm.sh/helm-v2.16.0-linux-amd64.tar.gz &&\
+  tar -xzf helm-v2.16.0-linux-amd64.tar.gz &&\
```

Comment:
Maybe you want to hoist these up into environment variables so they can be specified in the Dockerfile as ENV/ARGs? It'll keep them a bit more visible than burying them midway through the script.

Reply:
This will soon be deleted and replaced by the AMI... just needed this working again in order to deploy to the sandbox until that's ready.

Reply:
But I agree on the principle :)